From: Peter Krempa <pkrempa@redhat.com>
Date: Wed, 29 Aug 2012 14:29:43 +0200
Subject: Revert "security: Add DAC to security_drivers"

This reverts commit 9f9b7b85c9b422e8f4e813f3920bf8f433246a4a.

The DAC security driver needs special handling and extra parameters and
can't just be added to regular security drivers.
---
 src/security/security_driver.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/src/security/security_driver.c b/src/security/security_driver.c
index e6da220..f450a94 100644
--- a/src/security/security_driver.c
+++ b/src/security/security_driver.c
@@ -35,7 +35,6 @@
 # include "security_apparmor.h"
 #endif
 
-#include "security_dac.h"
 #include "security_nop.h"
 
 #define VIR_FROM_THIS VIR_FROM_SECURITY
@@ -47,7 +46,6 @@ static virSecurityDriverPtr security_drivers[] = {
 #ifdef WITH_SECDRIVER_APPARMOR
     &virAppArmorSecurityDriver,
 #endif
-    &virSecurityDriverDAC,
     &virSecurityDriverNop, /* Must always be last, since it will always probe */
 };
 
